VAMobile/src/components/VAButton.test.tsx

```diff
@@ -32,137 +22,81 @@ context('VAButton', () => {
         disabledText={'my button instructions'}
       />,
     )
-
-    testInstance = component.UNSAFE_root
   }
 
   beforeEach(() => {
     initializeTestInstance()
   })
 
-  it('initializes correctly', async () => {
-    expect(component).toBeTruthy()
+  it('initializes correctly', () => {
+    expect(screen.getByRole('button', { name: 'my button' })).toBeTruthy()
   })
 
-  it('should call onChange', async () => {
-    testInstance.findByType(VAButton).props.onPress()
+  it('should call onChange', () => {
+    fireEvent.press(screen.getByRole('button'))
     expect(onPressSpy).toBeCalled()
   })
 
   describe('when icon props are passed in', () => {
-    it('should render a VAIcon', async () => {
+    it('should render a VAIcon', () => {
       initializeTestInstance(false, ButtonTypesConstants.buttonPrimary, true)
-      expect(testInstance.findAllByType(VAIcon).length).toEqual(1)
+      expect(screen.UNSAFE_getAllByType(VAIcon).length).toEqual(1)
```

Comment:
you can just delete this test honestly, we are going to be converting this to the design systems team library button and this component will no longer be used at that point anyways in a few weeks

Reply:
in the conversion icons will no longer be on these buttons

Reply:
Removed all VAIcon tests